frontend/src/pages/videos/VideoAnalysisPage.tsx

```diff
@@ -39,112 +39,117 @@ export const VideoAnalysis = ({
-                        {t('entityAnalysisPage.chart.criteriaScores.title')}
-                      </Typography>
-                    </Box>
-                    <PersonalCriteriaScoresContextProvider uid={uid}>
-                      <Box px={2} pt={1}>
-                        <PersonalScoreCheckbox />
-                      </Box>
-                      <Box p={1}>
-                        <CriteriaBarChart video={video} />
-                      </Box>
-                    </PersonalCriteriaScoresContextProvider>
-                  </Paper>
-                </Grid>
-                <Grid item xs={12} sm={12} md={6}>
-                  <Paper>
-                    <Box
-                      p={1}
-                      bgcolor="rgb(238, 238, 238)"
-                      display="flex"
-                      justifyContent="center"
-                    >
-                      <Typography variant="h5">
-                        {t('criteriaScoresDistribution.title')}
-                      </Typography>
+                    p={1}
+                    bgcolor="rgb(238, 238, 238)"
+                    display="flex"
+                    justifyContent="center"
+                  >
+                    <Typography variant="h5">
+                      {t('entityAnalysisPage.chart.criteriaScores.title')}
+                    </Typography>
+                  </Box>
+                  <PersonalCriteriaScoresContextProvider uid={uid}>
+                    <Box px={2} pt={1}>
+                      <PersonalScoreCheckbox />
                     </Box>
                     <Box p={1}>
-                      <CriteriaScoresDistribution uid={uid} />
+                      <CriteriaBarChart video={video} />
                     </Box>
-                  </Paper>
-                </Grid>
-              </SelectedCriterionProvider>
-            )}
-          </Grid>
-        </Grid>
-        <Grid item md={4} sm={12}>
-          <ContextualRecommendations
-            contextUid={uid}
-            uploader={video.uploader || undefined}
-          />
+                  </PersonalCriteriaScoresContextProvider>
+                </Paper>
+              </Grid>
+              <Grid item xs={12} sm={12} md={6}>
+                <Paper>
+                  <Box
+                    p={1}
+                    bgcolor="rgb(238, 238, 238)"
+                    display="flex"
+                    justifyContent="center"
+                  >
+                    <Typography variant="h5">
+                      {t('criteriaScoresDistribution.title')}
+                    </Typography>
+                  </Box>
+                  <Box p={1}>
+                    <CriteriaScoresDistribution uid={uid} />
+                  </Box>
+                </Paper>
+              </Grid>
+            </SelectedCriterionProvider>
+          )}
         </Grid>
-      </Grid>
+      </Box>
+      <Box width={{ sm: '100%', xs: '100%', md: '33%', lg: '33%' }}>
```

Comment:
If any of us knows how to replace this verbose width by the proper usage of `flex-grow`, it would be great.

Reply:
A potential solution seems to be 
First Box (line 50): `<Box flex={2} minWidth={{ xs: '100%', md: null }}>`
Second Box (line 147): `<Box flex={1}>`

Reply:
Thank u, it works!